docs/nidigital/class.rst

```diff
@@ -5219,7 +5219,7 @@ specific_driver_vendor
 
     .. py:attribute:: specific_driver_vendor
 
-        Returns a string ("National Instruments") that contains the name of the vendor that supplies the NI-Digital Pattern Driver.
+        Returns a string ("NI") that contains the name of the vendor that supplies the NI-Digital Pattern Driver.
```

Comment:
We didn't update the driver to return `NI` as part of brand identity updates; it still returns `National Instruments`.

Reply:
Updated.